sdk/connect/javascript.md

```diff
@@ -39,23 +33,17 @@ import { MetaMaskSDK } from "@metamask/sdk"
 
 const MMSDK = new MetaMaskSDK({
   dappMetadata: {
-    name: "Example JavaScript Dapp",
+    name: "Example JavaScript dApp",
     url: window.location.href,
+    // iconUrl: 'https://mydapp.com/icon.png' // Optional
   },
   infuraAPIKey: process.env.INFURA_API_KEY,
 })
+```
 
-const ethereum = MMSDK.getProvider()
+- - `dappMetadata` ensures trust by showing your dapp’s `name`, `url`, and `iconUrl` during connection.
 
-// Connect to MetaMask
-const accounts = await MMSDK.connect()
-
-// Make requests
-const result = await ethereum.request({ 
-  method: "eth_accounts", 
-  params: [] 
-})
-```
+- - `infuraAPIKey` enables read-only RPC and load‑balancing.
```

Comment:
Can you move this bullet list outside of the `<Tabs>`? It should appear after Pure JS, too. Also, maybe preface with something like:
> These examples configure the SDK with the following options: